Map Update 19.07.2025

Corde50 opened the pull request:
# Описание
- Мелкие фиксы (Лестницы в роще друидов теперь видно с верхних Z-уровней;
- Дверь в "подсобку" таверны теперь открывается ключом тавернщика;
- Больше укромных мест в канализации.

Corde50 committed:
Map Update 1

- Мелкие фиксы.
- Дверь в "подсобку" таверны теперь открывается ключом тавернщика.
- Больше укромных мест в канализации.

SmiLeYre commented:
<!-- tgs_test_merge_comment -->
## Test merge deployment history:

<details><summary>Test Merge Deployed @ 07/18/2025 18:46:14 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: ff1570ee86aca78a6e1a92b160fd4d1b3b224d13
Pull Request: ec3940fc8565eac9bb574d44dd974186c4b35efa
Server: 966264497e7d0bd69d075b2a208ea694ef5d18fe
</details>
<details><summary>Test Merge Updated @ 07/19/2025 00:05:05 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: ff1570ee86aca78a6e1a92b160fd4d1b3b224d13
Pull Request: 0c16ba65805a36f1b0a910a7532f4cc16cc04f8e
Server: 545c131d74d31fc965aab0ba27b627ac8c25ac64
</details>
<details><summary>Test Merge Updated @ 07/20/2025 00:50:16 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: 3e0b187c73351a52f592741c352ddeec4b24916a
Pull Request: 0c16ba65805a36f1b0a910a7532f4cc16cc04f8e
Server: d3068cb92af0c3709abec2060e75f512f71d4f6e
</details>
<details><summary>Test Merge Updated @ 07/21/2025 08:14:34 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: 223f8a08b0a13432ce1eff049e2020b0b3984fc3
Pull Request: 0c16ba65805a36f1b0a910a7532f4cc16cc04f8e
Server: 1ec8fb022d6a17510f903f44eeba3b1a1a13df35
</details>
<details><summary>Test Merge Updated @ 07/21/2025 11:05:50 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: 828e89a2f2f9ed438d6966b483d31b809698384a
Pull Request: 0c16ba65805a36f1b0a910a7532f4cc16cc04f8e
Server: 0534d70073f25aa87d6b4217661bd7fede87258a
</details>
<details><summary>Test Merge Updated @ 07/22/2025 14:25:16 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: 828e89a2f2f9ed438d6966b483d31b809698384a
Pull Request: 706121716cb2be110abd8f672dede8d607f330dd
Server: 952fa82d98f03a7a96ff4c5240184b65cda0a7a4
</details>
<details><summary>Test Merge Updated @ 07/22/2025 22:48:59 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: 37260be480720da89f4064d21784f3c49df3a887
Pull Request: 706121716cb2be110abd8f672dede8d607f330dd
Server: 0b32e00f73dd7cddb3a02a078d7f576cbaafa3c3
</details>
<details><summary>Test Merge Updated @ 07/24/2025 00:19:09 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: ea501128393dbd12c18aac56aaa33baedf4216f4
Pull Request: 706121716cb2be110abd8f672dede8d607f330dd
Server: 903c49fb771507e6401f6db051f3cd560bff06f1
</details>
<details><summary>Test Merge Updated @ 07/24/2025 05:12:28 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: e849aa4bfddf2b069b803d78404387a71f1d2950
Pull Request: 706121716cb2be110abd8f672dede8d607f330dd
Server: 8fd2079903d36f7a5e8969d19697fa621fff23fe
</details>
<details><summary>Test Merge Updated @ 07/27/2025 18:35:36 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: e849aa4bfddf2b069b803d78404387a71f1d2950
Pull Request: e83fd8fc73f85cb4cec380b3d30f7fdb12985f47
Server: bf8528e292c3ff8411f8059798f28fc88f0279e0
</details>
<details><summary>Test Merge Updated @ 07/27/2025 20:30:48 +00:00:</summary>

##### Server Instance

[Redmoon-Keep](https://github.com/BlueMoon-Labs/Redmoon-Keep/deployments/activity_log?environment=TGS%3A+Redmoon-Keep)

##### Merged By
SmileYc0m

##### Revision
Origin: 1319043398efecb3e4a3f09a6865a66bc1978fba
Pull Request: e83fd8fc73f85cb4cec380b3d30f7fdb12985f47
Server: bab5aabdd4505062b058c96fb510c4881f423296
</details>
<details><summary>Test Merge Updated @ 08/08/2025 07:23:24 +00:00:</summary>

##### Server Instance
Moon-Red

##### Merged By
Admin

##### Revision
Origin: a35fa975e83c6aa6cc3baa18d31432d9cf47727f
Pull Request: e83fd8fc73f85cb4cec380b3d30f7fdb12985f47
Server: d35016b6f37b577c28d0cb7dc374715506b3410c
</details>

Corde50 committed:
Update 2

- Туман войны
- Немного изменен склеп
- Что-то еще там по мелочи.

Corde50 committed:
Map Update 3

- Манацветки, арканные артефакты и болотная трава вновь спавнится как прежде.
- Немного изменена планировка подземелий чертога, добавлен тайный причал.
- Заменил и добавил дома в деревне.
- Косметические улучшения в нескольких местах.
- Добавлено несколько точек интереса.
- Фиксы по мелочи.

Corde50 committed:
Update 4

- Туман теперь сплошная стена на 5 уровней без зазоров.

Corde50 committed:
Update 5

- Добавил дом ведьмы на болотах
- Добавил деревянную хижину в болотах
- Пофиксил тайные проходы в поместье, теперь они работают как должны
- Больше укромных и уединенных мест в канализации города
- Мелкие фиксы
